src/test/java/demo/selenium/test/heroku/LoginTest.java

```diff
@@ -1,4 +1,38 @@
 package demo.selenium.test.heroku;
 
-public class LoginTest {
+import demo.selenium.model.User;
+import demo.selenium.page.heroku.HerokuLoginPage;
+import demo.selenium.test.BaseTest;
+import org.jsoup.Connection;
+import org.junit.Assert;
+import org.junit.Test;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+
+public class LoginTest extends BaseTest {
+
+    @Test
+    public void positiveLoginTest() {
+        HerokuLoginPage herokuLoginPage = new HerokuLoginPage(driver);
+        User user = new User();
+        user.setUsername("tomsmith");
+        user.setPassword("SuperSecretPassword!");
+        herokuLoginPage.checkLogIn(user);
+        WebElement message = driver.findElement(By.id("flash"));
+        WebElement logout = driver.findElement(By.cssSelector(".secondary.radius"));
+        Assert.assertTrue("Login is unsuccessful",message.getText().contains("You logged into a secure area!"));
```

Comment:
Assert message should be something which is discribing error. Like: Notification message is not OK. Something which can tell you what is not OK just by reading it.